engines/config-query-sparql/config/function-factory/actors.json

```diff
@@ -3,6 +3,90 @@
     "https://linkedsoftwaredependencies.org/bundles/npm/@comunica/config-query-sparql/^3.0.0/components/context.jsonld"
   ],
   "import": [
+    "ccqs:config/function-factory/actors/term-function-xsd-to-year-month-duration.json",
```

Comment:
Could we get these sorted somehow?
Just so they are shown in the same order as when looking on the filesystem.